mon-pix/app/templates/components/qcm-proposals.hbs

```diff
@@ -6,7 +6,7 @@
          id="checkbox_{{inc index}}"
          checked={{labeledCheckbox.[1]}}
          name="{{inc index}}"
-         onclick={{action "checkboxClicked"}}
+         onclick={{action "checkboxClicked" (inc index)}}
```

Comment:
je suis surpris que le `onclick` fonctionne. D'après ma lecture de la doc [ici ("More events types are also supported…")](https://guides.emberjs.com/release/components/built-in-components/#toc_actions) et [ici (event handlers)](https://api.emberjs.com/ember/release/classes/Component#event-handler-methods), j'aurais tendance à mettre directement `click={{…}}`